js/dlg/message-box.js

```diff
@@ -102,6 +106,28 @@ messageBox.show = async ({
       scroll() {
         scrollTo(messageBox._blockScroll.x, messageBox._blockScroll.y);
       },
+      mouseDown(event) {
+        clickX = event.clientX;
```

Comment:
Add a check for `event.button === 0` because the user may [accidentally or intentionally] press other buttons while dragging.

Same for `mouseUp`